tests/system/HTTP/CURLRequestTest.php

```diff
@@ -1212,6 +1212,42 @@ public function testForceResolveIPUnknown(): void
         $this->assertSame(\CURL_IPRESOLVE_WHATEVER, $options[CURLOPT_IPRESOLVE]);
     }
 
+    public function testDNSCacheTimeoutEmptyOption(): void
+    {
+        $this->request->request('POST', '/post', [
+            'dns_cache_timeout' => 0,
```

Comment:
For completeness of tests, could you add a negative int time out? The PHP manual does not say anything about it being positive int so let's cover that case.

Reply:
https://linux.die.net/man/3/curl_easy_setopt

-1 is forever
0 for disabled